app/Http/Controllers/HeroController.php

```diff
@@ -88,7 +88,7 @@ public function show(){
           "description" => "Major patches",
           "conditional_field" => "timeframe_type",
           "conditional_value" => "major",
-          "options" => $global->convertToFilter(Session::get('all_major_patch')),
+          // "options" => $global->convertToFilter(Session::get('all_major_patch')),
```

Comment:
Hello! What's the reason why you are commenting these out and not removing them completely?